### PR TITLE
Update Specification

### PR DESCRIPTION
**Pull request by jmanideep**
I have updated the specification to clarify the properties each object type should have. Please review the changes and provide any feedback.

- **Dataset Objects (`@type: Dataset`)**
  - **Properties:** `name`, `author` (should include).
  - **Optional Properties:** `identifier`, `dateCreated`, `dateModified`, `text`, `keywords`, `comment`.
  - **Additional Files:** If a Dataset object contains additional files, they should be listed in its `hasPart` property and can be referenced through their `@id`.

- **File Objects (`@type: File`)**
  - **Properties:** `name`, `encodingFormat`, `contentSize` (should include).
  - **Optional Properties:** `description`, `sha256`, `author`, `identifier`, `dateCreated`, `dateModified`, `text`.

- **Organization Entities**
  - **Properties:** `@id`, `@type: Organization`, `name`, `url` (should include)
  - **Optional Properties:** Other properties from the `Organization`.

Please review the changes and let me know if you have any suggestions.

### CODE REVIEW

File: SPECIFICATION.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The rest of the archive is composed of 0 or more folders that each describe one
 
 Example for file: some-data.eln
 
-~~~yaml
+```yaml
```
Comment (NicolasCARPi) [inline SPECIFICATION.md:25]: what's wrong with using `~` ?

File: SPECIFICATION.md [hunk 2]
```diff
@@ -70,83 +70,99 @@ The first thing we do is describe the `ro-crate-metadata.json` object:
...
+```
 
-Its type is an array of `Dataset` and `hasPart` which corresponds to the differents `@id` of the other objects. Think of it like a Table of Contents.
+Its type is an array of `Dataset` and `hasPart` which corresponds to the different `@id`s of the other objects. Think of it like a Table of Contents.
 
 ### The rest
 
 After that, all the other objects have an `@type` of either `Dataset` for directories or `File` for well..., files. And the `@id` corresponds to something in the `hasPart` of `./`.
 
+If a Dataset object (node) has additional files, they should be listed in its `hasPart` property and can be referenced through their `@id`.
```
Comment (NicolasCARPi) [inline SPECIFICATION.md:112]: I suggest using exclusively **node** terminology. With a sentence explaining what is a `node` in the context of the JSON-LD graph.

File: SPECIFICATION.md (hunk 1 above)
Comment (jmanideep) [reply SPECIFICATION.md:25]: The formatting was automatically adjusted by Black.

Comment (NicolasCARPi) [reply SPECIFICATION.md:25]: Oh okay then, let's leave it like that. I prefer the `~` but let's go with the default choice that was made by black!

File: SPECIFICATION.md (hunk 2 above)
Comment (jmanideep) [reply SPECIFICATION.md]: Used node terminology wherever possible and also added a sentence explaining what a `node` is.

File: SPECIFICATION.md [hunk 3]
```diff
@@ -70,83 +70,99 @@ The first thing we do is describe the `ro-crate-metadata.json` object:
     "@id": "https://w3id.org/ro/crate/1.1"
   },
   "dateCreated": "2022-05-30T12:25:36+0200",
+  "sdPublisher": [
```
Comment (NicolasCARPi) [inline SPECIFICATION.md:73]: sdPublisher value is not an array, it should be:

~~~json
"sdPublisher": {"@id": "https://eln-example.com"},
~~~

**Review by NicolasCARPi — CHANGES_REQUESTED** [review]

**Review by NicolasCARPi — APPROVED** [review]

**Pull request merged**